modules/siddhi-extensions/time/src/test/java/org/wso2/siddhi/extension/time/DateFormatFunctionExtensionTestCase.java

```diff
@@ -385,4 +385,143 @@ public void receive(long timeStamp, Event[] inEvents, Event[] removeEvents) {
         Assert.assertTrue(eventArrived);
         executionPlanRuntime.shutdown();
     }
+
+    @Test(expected = ExecutionPlanValidationException.class)
+    public void dateFormatFunctionExtension11() throws InterruptedException {
+
+        log.info("DateFormatFunctionExtensionTestCaseInvalidParameterTypeFirstArgumentLenghtTwo");
+        SiddhiManager siddhiManager = new SiddhiManager();
+
+        String inStreamDefinition = "define stream inputStream (symbol string," +
+                "dateValue int,sourceFormat string,timestampInMilliseconds long,targetFormat string);";
+        String query = ("@info(name = 'query1') from inputStream select symbol , " +
+                "time:dateFormat(dateValue,targetFormat) as formattedDate," +
+                "time:dateFormat(timestampInMilliseconds," +
+                "targetFormat) as formattedUnixDate insert into outputStream;");
+        ExecutionPlanRuntime executionPlanRuntime = siddhiManager
+                .createExecutionPlanRuntime(inStreamDefinition + query);
+        executionPlanRuntime.start();
```

Comment:
let's remove executionplanRuntime start and shutdown in all attribute validation cases .Please make the change in others as well

Reply:
Ok,I will do it

Reply:
Fixed in 7693975